### ISSUE TITLE
TypeError: Cannot read property 'getHttpServer' of undefined

### ISSUE THREAD

**Issue by LeoMartinDev** [nestjs/nest#689]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Throw an unexpected error when running `npm run test:e2e`.

`TypeError: Cannot read property 'getHttpServer' of undefined`

## Expected behavior

It should run my tests.

## Minimal reproduction of the problem with instructions

```typescript
import { AppModule } from './../src/app.module';
import { INestApplication, Logger } from '@nestjs/common';
import * as request from 'supertest';
import { Test } from '@nestjs/testing';

describe('Test', () => {
  let app: INestApplication;

  beforeAll(async () => {
    const module = await Test.createTestingModule({
      imports: [AppModule],
    })
      .compile();

    app = module.createNestApplication();
    await app.init();
  });

  it(`/GET test`, () => {
    return request(app.getHttpServer())
      .get('/test')
      .expect(200);
  });

  afterAll(async () => {
    await app.close();
  });
});
```

## Environment

<pre><code>
Nest version: 5.0.0-beta.6

 
For Tooling issues:
- Node version: 9.9.0  
- Platform:  Windows

Others:

</code></pre>

**Comment by LeoMartinDev** [comment]
Solved, `@nest/testing` was stucked on 4.6.6, I updated directly my `package.json` to 5.0.0.

**Comment by Erkajp** [comment]
how to solve on 5.3.1 ?
